certified-connectors/Socialinsider/README.md

```diff
@@ -0,0 +1,16 @@
+# Socialinsider Analytics
```

Comment:
Hello @brospresident,

Could you please refer below template link and modify readme.md file?
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Hi @vmanoharas,

I modified the readme.md file using the template you provided.

Thank you!

Reply:
Thank you @brospresident for updating the readme.md file.